project_generators/msvs.py

```diff
@@ -767,6 +770,11 @@ def collect_source(self):
 
         # Try to include all existing files in the project
         self.source = self.ctx.srcnode.ant_glob('**', excl=exclude_files)
+
+        for srcnode in self.msvs_extend_sources:
+            print("Extend sources: {}".format(srcnode.ant_glob('**', excl=exclude_files)))
```

Comment:
Maybe we should kill this print. Converting an ant_glob to string prints all the items, right?
Now imagine that we extend sources for a massive project ;)

Reply:
Ah yes, the print should be removed - will do that